pkg/kube/controllers/extendedjob/errand_reconciler.go

```diff
@@ -99,3 +116,41 @@ func (r *ErrandReconciler) Reconcile(request reconcile.Request) (reconcile.Resul
 
 	return reconcile.Result{}, nil
 }
+
+// injectContainerEnv injects AZ info to container envs
+// errands always have an AZ_INDEX of 1
+func (r *ErrandReconciler) injectContainerEnv(podSpec *corev1.PodSpec) {
+
+	containers := []*corev1.Container{}
```

Comment:
Please, initialize the slice with capacity to avoid unnecessary memory copying:
```go
containers := make([]*corev1.Container, 0, len(podSpec.Containers) + len(podSpec.InitContainers))
```